### DISCUSSION TITLE
How to ignore delete events when listening to events

### DISCUSSION THREAD

**Discussion by GOOD-AN** [agronholm/apscheduler#730] [Q&A]
When I listen for events, I need to listen for delete, miss, run error and executed events at the same time, and each event will run a different function, but once a one-time task with a specified date is executed, the delete event will definitely be triggered before execution, which will lead to conflicts in subsequent runs of my program, how can I ignore the delete event due to execution?

**Comment by agronholm** [comment]
So am I understanding this correctly: You wish to differentiate between delete events caused by a job's schedule having run its course, and a job being explicitly deleted?

**Reply by GOOD-AN** [reply]
Yes, that's what I meant.

**Reply by agronholm** [reply]
Sorry, I had totally forgotten about this. I don't think you currently can distinguish between the two. You could request this as a feature though. Shouldn't be that hard to implement.

**Reply by GOOD-AN** [reply]
Okay, thank you very much.
